### PR TITLE
Add profile editing feature for employer

### PR DESCRIPTION
**Pull request by nvvu99**
## Related Tickets
- [#39182](https://edu-redmine.sun-asterisk.vn/issues/39182)

## WHAT (optional)
Add profile editing feature for employer

## HOW
Validate user input using form request.
Authorize user using policy.

## Evidence (Screenshot or Video)
* Edit profile
Employer edit profile

* View profile
Employer profile

### CODE REVIEW

**Review by vuminhhieu1311 — APPROVED** [review]

**Comment by vuminhhieu1311** [comment]
fix conflict đi anh

File: app/Http/Controllers/EmployerProfileController.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        return redirect()->route('employer.profiles.show', [
+            'profile' => $profile,
+        ]);
+    }
+
+    private function updateImageAttribute($request, $profile, $attributeName)
+    {
+        if ($request->hasFile($attributeName)) {
+            $photoPath = $request
+                ->file($attributeName)
```
[the remark this answers is not in the document]
Comment (nvvu99) [reply app/Http/Controllers/EmployerProfileController.php:69]: Laravel sẽ tự đổi tên file khi dùng hàm store.

[the remark this answers is not in the document]
Comment (vuminhhieu1311) [reply app/Http/Controllers/EmployerProfileController.php:69]: à ừ oki anh

**Review by vuminhhieu1311 — APPROVED** [review]
ready đi a

**Comment by nvvu99** [comment]
ready

**Comment by nvvu99** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**